DFC.FindACourseClient/Extensions/CourseTypeExtensions.cs

```diff
@@ -57,10 +57,6 @@ internal static List<DeliveryMode> MapToCompositeDeliveryModes(this IList<Comp.C
 
                     case Comp.CourseType.All:
                     default:
-                        deliveryModeList.Add(DeliveryMode.ClassroomBased);
```

Comment:
Is this correct to remove these? (and same with ones above)

Reply:
Yes its in because now, if none of the types has been picked on the frontend we are not defaulting to all of them we are leaving it blank so that it does not get sent as part of the request.